samples/apps/SmartLauncher/SmartLauncher.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
     <UserSecretsId>SmartLauncher_3d9536f4-982c-4334-83df-890fb8ae70e6</UserSecretsId>
+    <IsPackable>false</IsPackable>
+    <PackageId>Microsoft.Internal.SmartLauncher</PackageId>
```

Comment:
If we don't want to pack it, what's the point of specifying package name?

Reply:
True, if packed it should have *.internal.* but I don't think it needs to be packed at all. I'll confirm what happens in the build output and update